interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -666,6 +695,62 @@ namespace cling {
   }
 
   Interpreter::CompilationResult
+  Interpreter::codeComplete(const std::string& line, size_t& cursor,
+                            std::vector<std::string>& completions) const {
+
+    const char * const argV = "cling";
+    std::string llvmdir = this->getCI()->getHeaderSearchOpts().ResourceDir;
+    std::string extra_part = "/lib/clang/3.9.0";
```

Comment:
Yes, I don't like it either, but I found it in ROOT TMetaUtils but then the incremental parser adds this part "/lib/clang/3.9.0" manually, so I ended up with two of them..